app/views/gardens/_plantings.html.haml

```diff
@@ -0,0 +1,3 @@
+- unless plantings.empty?
+  - plantings.each do |p|
+    = link_to p.crop.name, p, :class => "label label-info", :title => "#{p.quantity} #{p.crop.name} #{p.planted_at}"
```

Comment:
I see why you've put the crop info in a title here so the plantings column won't be so cluttered. This is not a bad idea, but one alternative is a [Bootstrap tooltip](http://getbootstrap.com/javascript/#tooltips). I think they're slightly more responsive than titles. 

Before changing anything, it's probably a good idea to get input from someone who uses Growstuff on a regular basis, like @Skud or @maco